src/layers.js

```diff
@@ -0,0 +1,223 @@
+import { binarySearch, binarySearchIndex, binarySearchInsert } from './util.js';
+
+export class LayerManager {
+    // private properties don't work with reify
+    // @TODO: refactor once we migrated to ES6 modules, see https://github.com/opentypejs/opentype.js/pull/579
```

Comment:
even after migrating to another transpiler we will never get real "private" attribute since our code has to be dumbed-down to ES5 (or lower?) :neutral_face:

Reply:
Well, we might distribute different flavours in the future, and one of them could be a pure ES6 build.